### PR TITLE
974 websiteone setup problems

### LINKED ISSUE #974: WebsiteOne setup problems
I was trying the setup script for Website one. But it didn't worked out smoothly. As a workaround i did all the steps manually starting with installing recent version of nodejs using nvm and then installing necessary npm modules.
Also some configuration changes made for postgres role/user, like changing authentication from peer to trust for local database.
After the setup was complete, i tried running tests and following issues came up:-

**rake rspec**
```
Failures:

  1) dashboard/index.html.erb displays a tab view
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

  2) dashboard/index.html.erb render and populate map of users includes map legend
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

  3) dashboard/index.html.erb render and populate map of users includes map
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

  4) dashboard/index.html.erb render and populate map of users should eq 20
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

  5) dashboard/index.html.erb displays statistics:  10 Articles
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

  6) dashboard/index.html.erb displays statistics:  5 Projects
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

  7) dashboard/index.html.erb displays statistics:  40 Documents
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

  8) dashboard/index.html.erb displays statistics:  100 Members
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

  9) dashboard/index.html.erb displays statistics:  200 Scrum Minutes
     Failure/Error: Unable to find matching line from backtrace
     Capybara::Webkit::ConnectionError:
       /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/bin/webkit_server failed to start.
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:87:in `parse_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:93:in `discover_port'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:74:in `start_server'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/connection.rb:31:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit/driver.rb:14:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-webkit-1.8.0/lib/capybara/webkit.rb:15:in `block in <top (required)>'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `call'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:84:in `driver'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/session.rb:70:in `initialize'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `new'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:276:in `current_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara/dsl.rb:46:in `page'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:52:in `block in after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-2.6.2/lib/capybara.rb:306:in `using_session'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:51:in `after_failed_example'
     # /home/touchps/.rvm/gems/ruby-2.2.2@agileventures/gems/capybara-screenshot-1.0.9/lib/capybara-screenshot/rspec.rb:81:in `block (2 levels) in <top (required)>'

Finished in 1 minute 33.65 seconds (files took 14.25 seconds to load)
829 examples, 9 failures, 1 pending

Failed examples:

rspec ./spec/views/dashboard/index.html.erb_spec.rb:14 # dashboard/index.html.erb displays a tab view
rspec ./spec/views/dashboard/index.html.erb_spec.rb:71 # dashboard/index.html.erb render and populate map of users includes map legend
rspec ./spec/views/dashboard/index.html.erb_spec.rb:75 # dashboard/index.html.erb render and populate map of users includes map
rspec ./spec/views/dashboard/index.html.erb_spec.rb:69 # dashboard/index.html.erb render and populate map of users should eq 20
rspec ./spec/views/dashboard/index.html.erb_spec.rb:23 # dashboard/index.html.erb displays statistics:  10 Articles
rspec ./spec/views/dashboard/index.html.erb_spec.rb:30 # dashboard/index.html.erb displays statistics:  5 Projects
rspec ./spec/views/dashboard/index.html.erb_spec.rb:44 # dashboard/index.html.erb displays statistics:  40 Documents
rspec ./spec/views/dashboard/index.html.erb_spec.rb:37 # dashboard/index.html.erb displays statistics:  100 Members
rspec ./spec/views/dashboard/index.html.erb_spec.rb:51 # dashboard/index.html.erb displays statistics:  200 Scrum Minutes

Randomized with seed 24639
```

Jasmine tests passed without any issues, however cucumber test failed for following two features

```
Failing Scenarios:
cucumber features/charge_activity.feature:6 # Scenario: Charge a user
cucumber features/projects.feature:184 # Scenario: Editing Pitch content with Mercury Editor
Snapshots:-
```

### PR DESCRIPTION
**Pull request by touchps**
fixes #974 

- Manual instruction for ubuntu 14.04 
- Tested against fresh provisioned ubuntu 14.04 in aws.

### CODE REVIEW

**Comment by tansaku** [comment]
:shipit: from me

File: docs/ubuntu/ubuntu_14.04_manual_install_notes.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+echo "websiteone" > .ruby-gemset
+```
+
+You can use gemset name of your own choice.
+
+
+## Install bundle
+Just to be safe, install bundle gem first. Otherwise `bundle install` may fail.
+```
+gem install bundle
```
Comment (diraulo) [inline docs/ubuntu/ubuntu_14.04_manual_install_notes.md]: @touchps shouldn't this be `gem install bundler`

File: docs/ubuntu/ubuntu_14.04_manual_install_notes.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+echo "ruby-2.2.2" > .ruby-version
+rvm gemset create websiteone
+rvm use ruby-2.2.2@agileventures
+echo "websiteone" > .ruby-gemset
+```
+
+You can use gemset name of your own choice.
+
+
+## Install bundle
```
Comment (diraulo) [inline docs/ubuntu/ubuntu_14.04_manual_install_notes.md]: The gem is `Bundler`

**Comment by diraulo** [comment]
@touchps left some comments for you. Once you fix them I'll pull this in 😉

**Comment by touchps** [comment]
I checked my command, and i did use `gem install bundle`. I checked docs and seems like 
'gem install bundler' should be the way.

Will change the doc. Thanks for the comments.

**Pull request merged**